### DISCUSSION TITLE
Simulating transdural direct current simulation

### DISCUSSION THREAD

**Discussion by dfehring** [simnibs/simnibs#365] [8. Simulation workflows]
Hi everyone! 

I was wondering if it was possible to simulate transdural direct current simulation rather than transcranial direct current stimulation? 

Stimulation was administered transdurally via a craniotomy, therefore modification of bone conductivity is not appropriate. 

Thank you for any insight!

**Comment by bjornsig** [comment]
No, unfortunately we currently don't support simulation of implanted electrodes.
